src/styles/utils/_variables.scss

```
@@ -0,0 +1,9 @@
+$block-size: 100px;
+$gap: 1px;
+$week: "mon" 0, 
+  "tue" $block-size + $gap,
+  "wed" $block-size * 2 + $gap * 2,
+  "thu" $block-size * 3 + $gap * 3,
+  "fri" $block-size * 4 + $gap * 4,
+  "sat" $block-size * 5 + $gap * 5,
+  "sun" $block-size * 6 + $gap * 6;
```

it's better not provide calculation inside variables. Move this calculations to @each $name, $day in $week 
```suggestion
$week: 
  "mon" 0,
  "tue" 1,
  "wed" 2,
  "thu" 3,
  "fri" 4,
  "sat" 5,
  "sun" 6;
```